packages/bpk-component-progress/src/BpkProgress.scss

```diff
@@ -41,7 +41,7 @@
     transition: width $bpk-duration-base ease-in-out;
     border-radius: $bpk-border-radius-sm;
 
-    @include bpk-themeable-property(background-color, --bpk-progress-bar-fill-color, $bpk-color-green-500);
+    @include bpk-themeable-property(background-color, --bpk-progress-bar-fill-color, $bpk-color-sky-blue);
```

Comment:
I would have thought the new brand uses a darker blue... 🤔

Reply:
Ignore me - this is correct!

Reply:
No according to the theming 🤔 might check the designs though. I think it keeps in line with the spinner colour